### DISCUSSION TITLE
ch_PP-OCRv4_server检测不出灰烬的烬字

### DISCUSSION THREAD

**Discussion by 1822067490** [PaddlePaddle/PaddleOCR#13240] [Q&A] [answered]
#### 问题描述 / Problem Description
ch_PP-OCRv4_server模型检测不出灰烬的烬字，偶尔会识别成其他字，使用飞浆官网通用OCR体验也识别不出来

#### 运行环境 / Runtime Environment
- OS: win10
- Paddle: 2.6.1
- PaddleOCR: 2.7.0.0

#### 附件 / Appendix
![1 - 副本](https://github.com/PaddlePaddle/PaddleOCR/assets/44697764/39a9d5bf-636d-41ce-8dda-f0a19c08ae00)
![2 - 副本](https://github.com/PaddlePaddle/PaddleOCR/assets/44697764/2d3a8dac-64bf-4e73-9f79-d8eb2f4c0ea7)

**Comment by GreatV** [comment, marked as answer]
可能需要你在自己的场景下，重新训练一下模型。
